queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where name like '%mon';
+select * from animals where date_of_birth between '2016-01-01' and '2019-12-31';
+select * from animals where neutered = true and escape_attempts < 3;
+select date_of_birth from animals where name = 'Agumon' or name = 'Pikachu';
+select name, escape_attempts from animals where weight_kg > 10.5;
+select * from animals where neutered = true;
+select * from animals where name not like 'Gabumon';
```

Comment:
* Kindly fix this `select * from animals where name not like 'Gabumon';` query result as when i ran the query this is what i get 
![image](https://user-images.githubusercontent.com/52497246/177217387-774a007d-1186-4ef2-b229-5287f9998118.png)